### PR TITLE
KOGITO-2907 -  Create Image for the github-ci-bot to be used on aws o…

### PR DESCRIPTION
**Pull request by Kaitou786**
…penshift
See: https://issues.redhat.com/browse/KOGITO-2907
Signed-off-by: Tarun Khandelwal <tarkhand@redhat.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM registry.access.redhat.com/ubi8/ubi
+
+ENV node_version=v12.18.2
```
Comment (Ginxo) [inline Dockerfile:3]: `ENV` by `ARG` since `node_version` is not needed by the bot

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+RUN yum install xz python3 make -y
+
+RUN curl -L https://nodejs.org/dist/${node_version}/node-${node_version}-linux-x64.tar.xz  -o node-${node_version}-linux-x64.tar.xz && \
+    tar -C /root/ -xf node-${node_version}-linux-x64.tar.xz
+
+ENV PATH=/root/node-${node_version}-linux-x64/bin:$PATH
+
+COPY . /kie-github-ci-bot/
```
Comment (Ginxo) [inline Dockerfile:12]: I would do
```
RUN npm install -g yarn
COPY . /tmp/src/
RUN yarn global add "file:/tmp/src" && rm -rf /tmp/src

ENTRYPOINT [ "kie-github-ci-bot" ]
```

adding 
```
  "bin": {
    "kie-github-ci-bot": "./bin/main.js"
  }
```

File: Dockerfile (hunk 1 above)
Comment (Kaitou786) [reply Dockerfile:3]: We are installing node on the image, so extracted it here.

File: Dockerfile (hunk 2 above)
Comment (Kaitou786) [reply Dockerfile:12]: Didn't we remove yarn dependency from the project?, also how does `ENTRYPOINT [ "kie-github-ci-bot" ]` going to work?

Comment (Kaitou786) [reply Dockerfile:12]: I mean how is it going to start our bot server to start listening?

Comment (Ginxo) [reply Dockerfile:12]: `bin/main`should be adapted, or create another one. So please forget my comments, we will do it in another PR

**Pull request merged**